### DISCUSSION TITLE
[Esame 14 Luglio 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#364] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f7ae6ae4-6a85-454c-9b7b-33e0c6bd7496)

**Comment by Elia-Belli** [comment]
**(a)** $(20,28)=4|24,\ (21,30)=3|27,\ (12,39)=3|18\Rightarrow$ il sistema è compatibile ed equivalente a:

$$\begin{cases}5x\equiv 6(7)\\
7x\equiv 9(10)\\
4x\equiv 6(13)\end{cases}\to \begin{cases}x\equiv 4(7)\\
x\equiv 7(10)\\
x\equiv 8(13)\end{cases}$$

Utilizzando il Metodo di Sostituzione ottengo $x\equiv 347(910)$

**Reply by Alby7503** [reply]
L’inverso della terza congruenza si ottiene moltiplicando per 10 (40 - 39 = 1)

(6 * 10) % 13 = 8 (se non sbaglio)
